hedera-mirror-common/src/test/java/com/hedera/mirror/common/domain/DomainBuilder.java

```diff
@@ -549,6 +555,32 @@ public DomainWrapper<FileData, FileData.FileDataBuilder> fileData() {
         return new DomainWrapperImpl<>(builder, builder::build);
     }
 
+    private DomainWrapper<FallbackFee, FallbackFee.FallbackFeeBuilder<?, ?>> fallbackFee() {
+        var builder = FallbackFee.builder().amount(id()).denominatingTokenId(entityId(TOKEN));
+        return new DomainWrapperImpl<>(builder, builder::build);
```

Comment:
Only top level domain objects should have a DomainWrapper.
```suggestion
    private FallbackFee fallbackFee() {
        return FallbackFee.builder().amount(id()).denominatingTokenId(entityId(TOKEN));
```

Reply:
I overlooked that one, thanks I have now fixed it.